epsagon/events/greengrasssdk.py

```diff
@@ -0,0 +1,77 @@
+"""
+Greengrass events module.
+"""
+
+from __future__ import absolute_import
+import traceback
+from uuid import uuid4
+
+from epsagon.utils import add_data_if_needed
+from ..trace import trace_factory
+from ..event import BaseEvent
+
+
+class GreengrassEvent(BaseEvent):
```

Comment:
consider renaming to greengrass publish event

Reply:
👍